aslprep/utils/misc.py

```diff
@@ -771,3 +787,255 @@ def estimate_labeling_efficiency(metadata):
             labeleff *= BS_PULSE_EFF ** metadata.get("BackgroundSuppressionNumberPulses", 1)
 
     return labeleff
+
+
+def estimate_att_pcasl(deltam_arr, plds, lds, t1blood, t1tissue):
+    """Estimate arterial transit time using the weighted average method.
```

Comment:
@mattcieslak would you mind taking a look at the license info here?

Reply:
This is great!

Reply:
Thanks!